docs/source/configuration/environmentvariables.md

```diff
@@ -24,10 +26,23 @@ You could, for example, build your Volto application, then start it in productio
 yarn build && RAZZLE_API_PATH=https://plone.org yarn start:prod
 ```
 
+`process.env.HOST`: Represents the host or IP address on which a server should listen, default is port 0.0.0.0.
+
+`process.env.PORT`: Used to specify the port on which a web server should listen for incoming requests, default is port 3000.
+
+`process.env.RAZZLE_API_PATH`: Used to configure the API path for the application.
+
+`process.env.RAZZLE_PUBLIC_URL`: Used specify the base URL or path where static assets (such as images, stylesheets, and other resources) are hosted or served.
+
+`process.env.RAZZLE_DEV_PROXY_API_PATH`: Used during development to configure a proxy for API requests.
+
+`process.env.RAZZLE_INTERNAL_API_PATH`: Used to specify the path to an internal API that the server-rendered application should use.
+
+`process.env.RAZZLE_PROXY_REWRITE_TARGET`: Used to specify the target URL for a proxy server.
+
```

Comment:
All of these environment variables should follow the existing pattern as the glossary items below.

I also don't know whether the prefix `process.env.` is necessary. Perhaps a Volto core maintainer knows?

Finally, do these belong under this heading or another? Again, perhaps a Volto core maintainer knows?

Reply:
@stevepiercy @Manas-Kenge `process.env.` is not part of the name of the environment variable. It's how the variable is accessed in Node. It doesn't make sense to use it here, which is explaining variables that can be set, not how to write code that uses them.